transactions/MikoSea/MikoSeaMarket/createListing.json

```
@@ -0,0 +1,11 @@
+{
+  "arguments": [
+    "nftID",
+    "salePrice",
+    "nftversion"
+  ],
+  "messages": {
+      "en": "Create listing",
+      "ja": "Create listing"
```

If there is no Japanese translation, you can just keep the English one:
```suggestion
      "en": "Create listing"
```

---

Thanks, I updated